integration-cli/docker_cli_build_test.go

```diff
@@ -5660,3 +5660,434 @@ func (s *DockerSuite) TestBuildNullStringInAddCopyVolume(c *check.C) {
 	_, err = buildImageFromContext(name, ctx, true)
 	c.Assert(err, check.IsNil)
 }
+
+func (s *DockerSuite) TestBuildBuildTimeArg(c *check.C) {
+	imgName := "bldargtest"
+	envKey := "foo"
+	envVal := "bar"
+	args := []string{
+		"--build-arg", fmt.Sprintf("%s=%s", envKey, envVal),
+	}
+	dockerfile := fmt.Sprintf(`FROM busybox
+		ARG %s=
```

Comment:
I'm sorry but this just looks really weird from a UX perspective. ping @tiborvass 

If someone is defining an ARG w/o a default value, forcing people to add a "=" is really non-intuitive, borderline annoying.